src/dex/curve-v1/pools/3pool.ts

```diff
@@ -67,11 +67,11 @@ export class ThreePool extends CurvePool {
   }
 
   handleCoinTransfer(event: any, state: PoolState, log: Log): PoolState {
-    const from = event.args.src;
+    const from = event.args.from;
```

Comment:
I think this is not a correct change, the ABI we are using has `Transfer` event with `src` in it, not `from`.